src/fields/field_set.hh

```diff
@@ -111,8 +111,10 @@ public:
 	 *
 	 * Input::Type::Record SomEquation::input_type=
 	 * 		Record("SomeEquation","equation's description")
-	 * 		.declare_key("data",Input::Type::Array(EqData().make_field_descriptor_type()),"List of field descriptors.");
+	 * 		.declare_key("data",Input::Type::Array(EqData().make_field_descriptor_type().close()),"List of field descriptors.");
```

Comment:
In some cases (transport, unit tests) method declare_key is called on returned record. Declare_key can't be called on closed record.